composer.json

```diff
@@ -39,6 +39,7 @@
         "jetbrains/phpstorm-stubs": "^2019.2",
         "ocramius/package-versions": "^1.4|^1.5",
         "phpunit/phpunit": "^8.4|^9.0",
+        "serendipity_hq/vardumper-cli-to-html": "^1.0",
```

Comment:
Try `dump()`, it's inclued with Tracy

Reply:
Doesn't `dump()` conflicts with the one from Symfony's VarDumper component?

I wrote yesterday this small function to save the dumped result to an HTML file that makes easier the navigation across nodes.

Reply:
Anyway I will remove it once I will have solved the problem of finding a path to access the static method's docblocks and return type hints.

Reply:
> Doesn't dump() conflicts with the one from Symfony's VarDumper component?

It's possible. In that case we should have own `dump()` function that is loaded first and delegates to Tracy one.

Symfony is not good in CLI

Reply:
@TomasVotruba 
> Symfony is not good in CLI

Exactly: it is impossible to read in CLI.

So I created this small function that saves the dump to an HTML file and so you have the dump from the CLI, but as if it was triggered in an HTML page: this way you can open and close nodes, highlight same objects in different path, search in the dump, etc...